content/metadata.yaml

```diff
@@ -1162,3 +1162,22 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2021-04-05
+ -
```

Comment:
The content looks good, but something is off in the yaml file formatting.  I'll check it out and push a commit to change anything I find.